src/data/utility/Util.js

```diff
@@ -225,3 +227,30 @@ export const mergeArraysWithObjects = (a = [], b = [], prop, mergeProp) => {
 
   return updatedArray;
 };
+
+/**
+ * It converts timestamps to UTC based on timezone from user identity.
+ * timestamp, userIdentityId, identities
+ * @param {Number} timestamp datetime timestamp - moment in time
+ * @param {Number} userIdentityId identity id of the user
+ * @param {Array} identities collection of identities
+ * @return {Object} moment instance which will have all moment methods
+ * Example of usage:
+ * getUserUTCDateTime(
+ *  new Date.getTime(),
+ *  1,
+ *  [...identities...]
+ * ).format('YYYY-MM-DD HH:mm')
+ */
+
+export const getUserUTCDateTime = (timestamp, userIdentityId, identities) => {
+  if (timestamp && userIdentityId && identities) {
+    const timezone = get(
```

Comment:
@julianideamotive why do we ask user to provide identity object and id. Why not just pass the timezone and convert it? This way your function would be more OO.

Reply:
@maartenhemmes this is something which we talked about. I wanted to move the logic which finds the proper timezone to this method. Otherwise we would need to do some of the logic from here in the app everytime when we want to use this method. 

Later when we will have timezone setting, we could change this method to get only that timezone.